starknet_py/net/full_node_client.py

```diff
@@ -226,6 +226,13 @@ async def call_contract(
     async def send_transaction(
         self, transaction: InvokeFunction
     ) -> SentTransactionResponse:
+        if transaction.version == 0:
```

Comment:
This warning should probably be thrown when constructing the object with the given version, so we don't have to remember putting this warning everywhere we send the tx